cmd/tidb-dashboard/main.go

```diff
@@ -117,6 +118,7 @@ func NewCLIConfig() *DashboardCLIConfig {
 		pdEndPoint.Scheme = "https"
 	}
 	cfg.CoreConfig.PDEndPoint = pdEndPoint.String()
+	cfg.CoreConfig.Prefix = strings.TrimRight(cfg.CoreConfig.Prefix, "/")
```

Comment:
prefer to move this to CoreConfig itself to that logic can be shared between dashbaord cmd and PD cmd. It can be called like: 

```
CoreConfig.Normalize()
```